### ISSUE TITLE
Links are broken

### ISSUE THREAD

**Issue by Mimakur** [jgeisler0303/PIC16F1454_USB2Serial#2]
Links are broken. Any chance to get schematic for this project or help with my schematic? I looked and tried to copy something from Microchip 8bit mcu PIC Xpress Dev Board schematics. For example they use pic16f1454 in this board: https://www.microchip.com/en-us/development-tool/DM182027

Link to my pcb-design and schematics. https://oshwlab.com/mimakur/pic_proto

It's not a good design and there are some errors. Maybe someone could show me the errors i have made, if i have.

**Comment by jgeisler0303** [comment]
Which links do you refer to?
I'm not sure what kind of schematic you need. All the information how to connect the chip is in the README. I used the chip in this project: https://github.com/jgeisler0303/Frienduino so maybe these schematics help you to learn how to use it.

**Comment by Mimakur** [comment]
Thank you for your awesomely quick response. I was hoping that i could program for example 28-40pin dip pics with this 16f1454 and get rid of my pickit 2/3. And for the sake of simplicity use that same usb cable for powering target pic and other modules also. Should i even be able to do this with your hex file or did i understood something critically wrong?

Mplab is not showing option to use 1454 as a programmer, but maybe it's not even supposed to do so, i have no idea...?! I'm sorry if you think that these are stupid questions or ideas and i must admit that i know very little about programming. I just like to desing pcb:s and thought that this would be cool project.

In a nutshell my fantasy would be to do this kind of board, but for 8-40pin pic microcontrollers. https://www.microchip.com/en-us/development-tool/DM164136

Link i was refering to was this sky.geocities.jp/home_iwamoto/P16F145x/P16_L03.htm

**Comment by jgeisler0303** [comment]
For what it's worth I updated the link with the internet archive version but I didn't get that to translate with google.

Regarding the reuse of the code with a different chip I'm afraid, I cannot help much. I also only hacked the project together form the original source without bothering too much to learn how it all really works. I usually don't work with PICs. So, I sorry but I can't help you here.

**Comment by Mimakur** [comment]
Okay. Thank you. Because of my enthusiams i somehow thought that this would work like a charm, out of the box with pics. I'm just beginning my journey with mcus. Thank you very much. Have a nice weekend.
